generators/client/templates/angular/src/main/webapp/app/shared/login/login.component.html.ejs

```diff
@@ -24,9 +24,7 @@
 <div class="modal-body">
     <div class="row justify-content-center">
         <div class="col-md-8">
-            <div class="alert alert-danger" *ngIf="authenticationError" jhiTranslate="login.messages.error.authentication">
-                <strong>Failed to sign in!</strong> Please check your credentials and try again.
-            </div>
+            <jhi-alert-error [timeout]=0></jhi-alert-error>
```

Comment:
why is timeout 0?

Reply:
Currently, in the login modal the alert message doesn't desappear so I keep the same behaviour by adding timeout = 0.